CMake/CopyVcpkgDlls.cmake

```diff
@@ -0,0 +1,16 @@
+# Add an extra step to copy built DLLs on MSVC
+IF (MSVC AND USE_VCPKG_DEPS)
+# Have to do this in two custom steps as we have some DLLs in a top level
+# and some in a debug folder so the $<CONFIGURATION> flag won't help
+
+    add_custom_command(TARGET Copy_Release_DLLS CorsixTH 
```

Comment:
Haven't tried it yet, but these don't look right: https://cmake.org/cmake/help/v3.2/command/add_custom_command.html#build-events

Disclaimer, I haven't tested it, but I think it should read like:
```
add_custom_command(
  TARGET CorsixTH POST_BUILD
  COMMAND ${CMAKE_COMMAND} -E copy_directory
    "${VCPKG_INSTALLED_PATH}/<$CONFIGURATION:Debug:debug/>bin"
    $<TARGET_FILE_DIR:CorsixTH>)
```